core/src/main/java/org/elasticsearch/cluster/routing/RoutingService.java

```diff
@@ -62,6 +64,10 @@
     private volatile boolean routingTableDirty = false;
 
     private volatile Future scheduledRoutingTableFuture;
```

Comment:
we don't do reroute every 10 seconds, we only do it if its dirty within this class